src/brasil/gov/tiles/tiles/templates/header.pt

```diff
@@ -9,21 +9,34 @@
     <tal:fields repeat="field view/get_configured_fields">
 
         <tal:link define="htmltag python:field.get('htmltag', 'h1')"
```

Comment:
tu te importa em corrigir a endentação deste arquivo completa? tem que usar 2 espaços em cada nível e 4 em linhas que listam atributos, como definido aqui:

http://docs.zope.org/zopetoolkit/codingstyle/zcml-style.html#tabs-and-spaces